src/Component/Molecule/Bubble.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import styled from 'styled-components';
+
+import { useRecoilState } from 'recoil';
+import { usersState } from '../../State/Users'
+
+import SquareImage from '../Atom/SquareImage';
+
+type Bubbleprops = {
+    senderId : string
+    receiverId : string
+    roomId : string
+    messageText : string
+}
+
+type SenderNameprops = {
+    receiverId : string
+    roomId : string 
+}
+
+const Bubble = ( { senderId, receiverId, roomId, messageText } : Bubbleprops ) => {
+    const [users , setUsers] = useRecoilState(usersState); 
+    const filteredUser = users.filter(user => user.id == senderId);
```

Comment:
`===`와 달리 `==`는 강제 type casting(형변환)을 적용하여 값을 비교하므로 [strict equality operator](https://developer.mozilla.org/ko/docs/Web/JavaScript/Equality_comparisons_and_sameness)의 사용을 지향합니다

```suggestion
    const filteredUser = users.filter((user) => user.id === senderId);
```